webapp/locales/de.json

```diff
@@ -21,6 +21,7 @@
     }
   },
   "site": {
+    "thx": "Danke!",
```

Comment:
No abbreviations please and be a little more verbose on the confirmation message.